poster/styles/styles.css

```diff
@@ -0,0 +1,293 @@
+body {
+  background-color: #FFC9B5;
+}
+
+.background-image {
+  right: 20px;
+  top: 100px;
+  width: 700px;
+  position: fixed;
+}
+
+.background-image-container {
+  position: relative;
+}
+
+.info-secondary {
+  position: fixed;
+  bottom: 80px;
+  right: 500px;
+  color: #FFFFFF;
+  font-family: "brevia", sans-serif;
```

Comment:
You could set the `body` font family with this rule, then you wouldn't need to repeat it here and on the `info` class. Same thing with the `color: #FFFFFF` declaration. Then, there would only be 2 rules that override it (for the header classes).